x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/service.ts

```diff
@@ -54,7 +54,7 @@ export const getTrustedAppsList = async (
     perPage,
     filter: undefined,
     namespaceType: 'agnostic',
-    sortField: 'name',
+    sortField: 'name.keyword',
```

Comment:
This little stinker caused me great pain 😄  I tried searching for any other instances where we were sorting exceptions by `name` and it seems this was the only one.

Reply:
Is this the way we want it? If you flip it back the other way with keyword first, `.text` secondary per my comment here:
https://github.com/elastic/kibana/pull/88701/files#r571107432

Then this would revert back the other way I'm thinking.

Reply:
👍  great catch. I know we spoke offline, but for anyone following - we may need some patches to SO _find api. Will update more once I sync with Rudolf.

Reply:
PR for SO _find api is open https://github.com/elastic/kibana/pull/90988